### PR TITLE
Add ltdetrv2-s/m/l/x aliases for EdgeCrafter LT-DETR models

### PR DESCRIPTION
**Pull request by gabrielfruet**
## What has changed and why?

Now we can resolve LTDETRv2 model names via the `ltdetrv2` alias. It is listed as available models.

## Did you update [CHANGELOG.md](../CHANGELOG.md)?

- [ ] Yes
- [ ] Not needed (internal change)

## Did you update the documentation?

- [X] Yes
- [ ] Not needed (internal change without effects for user)

### CODE REVIEW

**Comment by gabrielfruet** [comment]
follow-up: https://github.com/lightly-ai/lightly-train/pull/798

File: docs/source/object_detection.md [hunk 1]
```diff
@@ -5,19 +5,20 @@
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/lightly-ai/lightly-train/blob/main/examples/notebooks/object_detection.ipynb)
 
 ```{note}
-🔥 LightlyTrain now supports training **LTDETR**: **DINOv3**-, **DINOv2**-, and **EdgeCrafter ECViT**-based object detection models with the super fast RT-DETR detection architecture! Our largest model achieves an mAP<sub>50:95</sub> of 60.0 on the COCO validation set!
+🔥 LightlyTrain now supports training **LTDETR**, a DINOv3-based object detection family with the super fast RT-DETR detection architecture! Pick a compact **LTDETRv2** model (`ltdetrv2-s/m/l/x`) for efficient detection, or a larger DINOv3 ViT/ConvNeXt model for maximum accuracy. Our largest model achieves an mAP<sub>50:95</sub> of 60.0 on the COCO validation set!
```
Comment (yutong-xiang-97) [inline docs/source/object_detection.md]: I would simply focus on promoting v2 and use LTDETRv1 only for comparison. For example:

```suggestion
🔥 LightlyTrain's **LTDETRv2** is out with great improvements in the supporting backbone and other contributions from SOTA research! We achieved 50.4mAP<sub>50:95</sub> on COCO 2017 validation set (+1 mAP from the previous LTDETR with 55% shorter training schedule). We also achieved 5.4ms latency on an NVIDIA T4 using TensorRT, FP16, batch size 1, and input resolution 640x640!
```

**Review by yutong-xiang-97 — APPROVED** [review]
